sycl/test-e2e/lit.cfg.py

```diff
@@ -583,19 +583,35 @@ def open_check_file(file_name):
 if platform.system() == "Linux":
     xptifw_dispatcher = os.path.join(xptifw_lib_dir, "libxptifw.so")
 elif platform.system() == "Windows":
-    xptifw_dispatcher = os.path.join(config.dpcpp_root_dir, "bin", "xptifw.dll")
+    # Use debug version of xptifw library if tests are built \MDd.
+    xptifw_dispatcher_name = ""
+    if "/MDd" in config.cxx_flags:
+        xptifw_dispatcher_name = "xptifwd.dll"
+    else:
+        xptifw_dispatcher_name = "xptifw.dll"
+
+    xptifw_dispatcher = os.path.join(
+        config.dpcpp_root_dir, "bin", xptifw_dispatcher_name
+    )
 xptifw_includes = os.path.join(config.dpcpp_root_dir, "include")
 if os.path.exists(xptifw_lib_dir) and os.path.exists(
     os.path.join(xptifw_includes, "xpti", "xpti_trace_framework.h")
 ):
     config.available_features.add("xptifw")
     config.substitutions.append(("%xptifw_dispatcher", xptifw_dispatcher))
     if cl_options:
-        xptifw_lib_name = os.path.normpath(os.path.join(xptifw_lib_dir, "xptifw.lib"))
+        # Use debug version of xptifw library if tests are built \MDd.
+        xptifw_lib_name = ""
+        if "/MDd" in config.cxx_flags:
+            xptifw_lib_name = "xptifwd.lib"
+        else:
+            xptifw_lib_name = "xptifw.lib"
```

Comment:
Can we move these names (without extensions) to some constant variables, because above on lines 589-591 there is `xptifw_dispatcher_name` variable with value which has the same names

Reply:
Yes, fixed per your suggestion.